### ISSUE TITLE
status function should return state and not boolean

### ISSUE THREAD

**Issue by phracek** [fedora-modularity/meta-test-family#168]
In PR https://github.com/fedora-modularity/meta-test-family/pull/164 we have discovered that status function returns boolean value.

Can we discuss it, whether we can return strings like "Running", "Failed", etc.?

Does it make sense at all?

**Comment by TomasTomecek** [comment]
Just return whatever is in metadata.

**Comment by jscotka** [comment]
I'm not sure. I prefer to return boolean, because interpretation has to be same for all backends.
In case it does sense, there could be more states, something like some interpretation table for all types, but return just what is in metadata is problematic in case it is not same for systemd-nspawn, docker, openshift, ... etc

**Comment by TomasTomecek** [comment]
Let me rephrase. This is the definition of the function:
```python
    def status(self):
        """
        Function returns whether the application exists
        and is Running in OpenShift environment

        :return: True application exists
                 False application does not exist.
        """
```

The dictionaries define status as:

    the situation at a particular time during a process

or

    state or condition of affairs

I understand that definition of the word status is an enumeration of states. @jscotka you suggest that the enumeration should hold just two states: true and false. That's perfectly fine. But in my world, that implementation doesn't fit the `status` method.

So what I had in mind is:

1. either change the name and documentation of the method (e.g. `def app_exists(self):`)
2. or change the implementation

because the current state is confusing. And ideally use the python way of doing things: [properties](https://docs.python.org/3/library/functions.html#property)
